### PR TITLE
Initialize USERNS_HOST variable

### LINKED ISSUE #2175: Scope script is broken for old docker versions
Introduced by https://github.com/weaveworks/scope/pull/2161 . Detected by @fbarl  on https://circleci.com/gh/weaveworks/scope/6274

### PR DESCRIPTION
**Pull request by 2opremio**
Fixes #2175

### CODE REVIEW

File: scope [hunk 1]
```diff
@@ -27,6 +27,7 @@ WEAVESCOPE_DOCKER_ARGS=${WEAVESCOPE_DOCKER_ARGS:-}
 #  "Cannot share the host's network namespace when user namespaces are enabled" for `docker run --net=host`
 # To avoid above errors, use `--userns=host` option to let container use host User Namespace.
 # This option(saved in $USERNS_HOST) will be inserted ONLY IF docker support `--userns` option.
+USERNS_HOST=""
```
Comment (rade) [inline scope]:
```
USERNS_HOST=
```
should suffice

Comment (rade) [inline scope]: but meh; looking at the weave net script, we use both forms there.

**Comment by rade** [comment]
LGTM, provided you have actually run this on an old and new docker.

**Comment by 2opremio** [comment]
> provided you have actually run this on an old and new docker.

Yep, I've run it in Circle

**Pull request merged**